include/odp/api/spec/timer.h

```diff
@@ -216,6 +216,23 @@ typedef struct {
 	 */
 	odp_timer_res_capability_t max_tmo;
 
+	/**
+	 * Enqueue queue type supported : Schedule queue.
+	 *
+	 * Defines the odp_queue_t type that the timer is capable of
+	 * delevering events to.
+	 * When set odp_timer_alloc() can be called with sched queue.
+	 */
+	odp_bool_t sched_queue_enq;
```

Comment:
Timer API uses term "destination queue for timeout". Both of these capabilities should refer to "destination queue" term. For example:
```
Scheduled destination queue support
0: Scheduled queues are not supported as timeout destination queues
1: Scheduled queues are supported as timeout destination queues
@see odp_timer_alloc()
odp_bool_t queue_type_sched;
```

Reply:
Sure, will send next version.
Thanks.